packages/compiler/core/binder.ts

```diff
@@ -93,30 +99,38 @@ export function createBinder(program: Program, options: BinderOptions = {}): Bin
     const rootNs = sourceFile.esmExports["namespace"];
     const namespaces = new Set<NamespaceStatementNode>();
     for (const [key, member] of Object.entries(sourceFile.esmExports)) {
-      if (typeof member === "function" && isFunctionName(key)) {
+      let name: string;
+      let kind: "decorator" | "function";
+
+      if (typeof member === "function") {
         // lots of 'any' casts here because control flow narrowing `member` to Function
         // isn't particularly useful it turns out.
-
```

Comment:
The changes in this part bind non-decorator exported functions so they can be called from projections.